doc/classes/CodeEdit.xml

```diff
@@ -421,7 +421,8 @@
 			<return type="void" />
 			<param index="0" name="force" type="bool" default="false" />
 			<description>
-				Emits [signal code_completion_requested], if [param force] is [code]true[/code] will bypass all checks. Otherwise will check that the caret is in a word or in front of a prefix. Will ignore the request if all current options are of type file path, node path, or signal.
+				If [method _request_code_completion] is overridden, invokes [method _request_code_completion] with [param force]. Otherwise emits [signal code_completion_requested] if either [param force] is [code]true[/code], or the caret is in word or after one of the [member code_completion_prefixes].
+				[b]Note:[/b] [signal code_completion_requested] is not emitted if all current options are any of [constant KIND_SIGNAL], [constant KIND_FILE_PATH] or [constant KIND_NODE_PATH]; irrespective of [param force].
```

Comment:
```suggestion
				[b]Note:[/b] [signal code_completion_requested] is not emitted if all current options are any of [constant KIND_SIGNAL], [constant KIND_FILE_PATH], or [constant KIND_NODE_PATH]; regardless of [param force].
```
Please use [Oxford comma](https://contributing.godotengine.org/en/latest/documentation/guidelines/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything). Also I'd say "irrespective" is not as fitting here